docs/docsource/operation/Upgrading_XDM.rst

```diff
@@ -0,0 +1,28 @@
+.. _Upgrading XDM:
+
+Upgrading XDM
+=============
+
+#. Install the new ISO with:
+   
+   .. code::
+      
+      curl -u '$CREDS' https://eve.devsca.com/github/scality/zenko/artifacts/builds/github:scality:zenko:staging-2.0.0.r210210230152.7c4bbdc.pre-merge.00015938/zenko-2.0.0-alpha.3.iso -o zenko-2.0.0-alpha.3.3.iso
+
+#. Import the ISO with:
+
+   .. code::
+      
+      /srv/scality/metalk8s-2.6.0/solutions.sh import --archive /home/centos/zenko-2.0.0-alpha.3.3.iso
```

Comment:
The current MK8s release is 2.7.1. Ask the SME if we want to keep documenting with 2.6.0. Same with two instances below (lines 22 and 28)

Reply:
I asked @thomasdanan about that

Reply:
You might want to ask someone on Zenko this question. It's possible that Zenko is tested on 2.6.0 and not on 2.7.1. It is also possible that it can update with no problem every time we update Zenko.